azure-quantum/pytest.ini

```diff
@@ -11,3 +11,4 @@ markers =
     quantinuum: mark a test as requiring access to quantinuum.
     rigetti: mark a test as requiring access to Rigetti targets.
     qci: mark a test as requiring access to QCI targets.
+    microsoft_qc: mark a test as requiring access to microsoft-qc targets.
```

Comment:
Is the underscore because dashes do not work for pytest marks?

Reply:
Yes, exactly.